scripts/release_scripts/initial_release_prep.py

```diff
@@ -141,7 +119,7 @@ def cut_release_branch():
     subprocess.check_call([
         'python', '-m',
         'scripts.release_scripts.cut_release_or_hotfix_branch',
-        '--new_version=%s' % release_version])
+        '--release_version=%s' % release_version])
```

Comment:
I just wanted to check why this was not caught on the test? Can we do something here, so that this can be caught in the future?

Reply:
We can update the tests to match the exact args but in that case too we will need to change it at both the places (the original file & test). Do you have any suggestions. on how to check this?

Reply:
I was expecting a test for `cut_release_branch` method which will run this script and fails if the `subprocess.check_call` for the `cut_release_or_hotfix_branch` throws error. do we have any such test?

Reply:
Basically when `cut_release_or_hotfix_branch` script is run through `subprocess.check_call`, the function which will be called is `execute_branch_cut`. We do not want that function to actually execute a branch cut during the test. So, I tried to mock it but mocking does not work since I import the `cut_release_or_hotfix_branch` and then mock the function but the script is being called through subprocess (and not being imported in the `initial_release_prep` script). 
So, to test this we need to mock the execute branch cut function (Note even if we mock the function and pass a wrong argument like `new_version` instead of `release_version`, there will be error in test and the test will then be able to catch such errors). I am not sure how to do this. Do you have any suggestions?

Reply:
@ankita240796 is there any specific reason that we will have to use `subprocess.check_call` to execute `execute_branch_cut`, can't we import the cut_release_or_hotfix_branch and call execute_branch_cut, with required params?

(Currently, `execute_branch_cut` is handling the params passed to the script which doesn't fit well with the function name. I think the main function should handle the params and call `execute_branch_cut` with the required params.)

Reply:
Yes, I think we can do this. Let me try this & get back to you. Thanks!

Reply:
Added a fix here: https://github.com/oppia/oppia/pull/8670/commits/4dafbc79a393499ad96b6e9c9a5a20b91f174cd1

Reply:
Thanks!

[I'm removing assignees from this PR, You can reply on the new PR if you have anything to add.]